robots-ev3-module/src/main/java/orwell/tank/SimpleEscapeKeyListener.java

```diff
@@ -7,9 +7,9 @@
 import org.slf4j.LoggerFactory;
 
 public class SimpleEscapeKeyListener implements KeyListener {
-    private final static Logger logback = LoggerFactory.getLogger(SimpleEscapeKeyListener.class);
+    private static final Logger logback = LoggerFactory.getLogger(SimpleEscapeKeyListener.class);
 
-    private boolean wasKeyPressed = false;
+    private boolean wasKeyPressed;
```

Comment:
It is probably better to have the value initialized to false.

Reply:
Ok!